### ISSUE TITLE
Don't require presence of Proxy in production

### ISSUE THREAD

**Issue by thomasboyt** [tbranyen/redux-create-action-types#5]
Proxy isn't polyfillable and isn't actually used in production, so it'd be nice to not require it to be present:

```js
// instead of
const TYPES = inProduction ? {} : new Proxy({}, handler);

// use something like
let TYPES = {};
if (inProduction && window.Proxy) {
  TYPES = new Proxy({}, handler);
}
```

**Comment by tbranyen** [comment]
I really like this idea, would you be interested in providing a PR?

**Comment by tbranyen** [comment]
Hrm, the more I think about this. You shouldn't ship the Proxy-code-path in production right?

**Comment by tbranyen** [comment]
Fixed in: 2.1.0
